token/services/selector/simple/service.go

```diff
@@ -33,11 +29,19 @@ type SelectorService struct {
 	managerLazyCache utils.LazyProvider[*token.ManagementService, token.SelectorManager]
 }
 
-func NewService(lockerProvider LockerProvider) *SelectorService {
+func NewService(tms *token.ManagementService, lockerProvider LockerProvider, cfg viewdriver.ConfigService) *SelectorService {
+	retryInterval := 5 * time.Second
```

Comment:
Maybe we could have a struct with this config that we unmarshal
```
type SelectorConfig struct {
 RetryInterval time.Duration
 NumRetries int
}

func (c *SelectorConfig) GetNumRetries() int {
 if c.NumRetries > 0 return c.NumRetries else return 3
}
```

Reply:
Hi @arner , did you address this?

Reply:
Yes @adecaro I rebased and added the SelectorConfig.